modules/cluster-cloud-controller-config-osp.adoc

```diff
@@ -0,0 +1,324 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-cloud-config-reference.adoc
+
+:_content-type: REFERENCE
+[id="cluster-cloud-controller-config_{context}"]
+= The {rh-openstack} Cloud Controller Manager (CCM) config map
+
+A {rh-openstack-first} CCM config map defines how your cluster interacts with your {rh-openstack} cloud. By default, this configuration is stored under the `cloud.conf` key in the `cloud-conf` config map in the `openshift-cloud-controller-manager` namespace. For example:
```

Comment:
```suggestion
A {rh-openstack-first} CCM config map defines how your cluster interacts with your {rh-openstack} cloud. By default, this configuration is stored under the `cloud.conf` key in the `openshift-cloud-controller-manager` config map in the `cloud-conf` namespace. For example:
```

@rlobillo Is this what you indicated in your comment? 

If so, I need to swap out similar language in #52520. 

I want to be sure, as that other QEed PR and the example config map that I found both have `openshift-cloud-controller-manager` as the namespace. 

@MaysaMacedo @stephenfin

Reply:
@maxwelldb Nop, this change is not correct. Please discard it.